build-scripts/cmake-toolchain-mingw64-i686.cmake

```diff
@@ -0,0 +1,13 @@
+set(CMAKE_SYSTEM_NAME Windows)
+set(CMAKE_SYSTEM_PROCESSOR x86)
+
+find_program(CMAKE_C_COMPILER NAMES i686-w64-mingw32-gcc)
+find_program(CMAKE_CXX_COMPILER NAMES i686-w64-mingw32-g++)
```

Comment:
You also want a `CMAKE_RC_COMPILER` (same for 64 bit version.)

Reply:
Interesting, I didn't end up needing that for some reason. @madebr?

Reply:
Me neither.
But I guess it's a fail safe such that people know `CMAKE_RC_COMPILER` is needed for "something".
It's not an error to add it.

Reply:
Is cmake finding it by itself based on a predicted prefix from the C compiler, or something?

Reply:
> `CMAKE_RC_COMPILER` is needed for "something"

we need it for building version.rc, do we not?

Reply:
Yes, it's for `version.rc`. (I was saying it in perspective of SDL users)

CMake sets a [default `CMAKE_RC_COMPILER` here](https://github.com/Kitware/CMake/blob/93ded7304aea9848352ce843d98fbf64023be11f/Modules/Platform/Windows-GNU.cmake#L159-L161).

The [cross compiler prefix is guessed here](https://github.com/Kitware/CMake/blob/93ded7304aea9848352ce843d98fbf64023be11f/Modules/CMakeDetermineCCompiler.cmake#L154-L193).

Reply:
OK. (Can't test it myself though -- my cmake is 3.9.6, mingw is made to require 3.13+)